lucene/core/src/test/org/apache/lucene/index/TestExitableDirectoryReader.java

```diff
@@ -160,5 +160,78 @@ public void testExitableFilterIndexReader() throws Exception {
 
     directory.close();
   }
+
+  /**
+   * Tests timing out of PointValues queries
+   *
+   * @throws Exception on error
+   */
+  @Ignore("this test relies on wall clock time and sometimes false fails")
```

Comment:
I wonder if we could make a mock clock and then have deterministic control and be able to re-enable these tests?

Reply:
I'll try to do so, yes, thanks :+1:

Reply:
Fixed in 2b3dfbcba8944dd48c22c341699da528c8525f91.